src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -283,11 +285,12 @@ public Attachment getAttachment(
         );
     }
 
-    private Links createLinks(String attachmentsUri, String attachmentId) {
-        String linkToSelf = attachmentsUri + "/" + attachmentId;
+    private Links createLinks(String selfLink, boolean download) {
```

Comment:
Minor but could rename second var to 'createDownloadLink' which would improve readability in the 'if' statement lower down.